yarn.lock

```diff
@@ -13990,7 +13985,7 @@ zen-observable-ts@^0.8.21:
     tslib "^1.9.3"
     zen-observable "^0.8.0"
 
-zen-observable@^0.8.0:
+zen-observable@^0.8.0, zen-observable@^0.8.15:
```

Comment:
We still need it 🤷‍♂️
```
yarn why memory-fs
yarn why v1.22.0
[1/4] 🤔  Why do we have the module "memory-fs"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "memory-fs@0.4.1"
info Has been hoisted to "memory-fs"
info Reasons this module exists
   - "workspace-aggregator-a1a81eac-55c3-4d63-804c-42926c9624f7" depends on it
   - Hoisted from "_project_#hops-webpack#webpack-dev-middleware#memory-fs"
   - Hoisted from "_project_#hops-postcss#webpack#memory-fs"
info Disk size without dependencies: "44KB"
info Disk size with unique dependencies: "344KB"
info Disk size with transitive dependencies: "580KB"
info Number of shared dependencies: 8
=> Found "enhanced-resolve#memory-fs@0.5.0"
info This module exists because "_project_#hops-bootstrap#enhanced-resolve" depends on it.
info Disk size without dependencies: "52KB"
info Disk size with unique dependencies: "352KB"
info Disk size with transitive dependencies: "588KB"
info Number of shared dependencies: 8
✨  Done in 2.89s.
```